docs/SMCWG-charter.md

```diff
@@ -0,0 +1,328 @@
+
+1. An Applicant not already a member of the Forum SHALL provide the following
+   information:
+
+   * Confirmation that the applicant satisfies at least one (1) of the
+     membership eligibility criteria (and if it satisfies more than one (1),
+     indication of the single category under which the applicant wishes to apply).
+   * The organization name, as they wish it to appear on the Forum Web site and
+     in official Forum documents.
+   * URL of the applicant’s main Web site.
+   * Names and email addresses of employees who will participate in the Working
+     Group and Forum as Member representatives.
+   * Emergency contact information for security issues related to certificate
+     trust.
+
+   Applicants that qualify as Certificate Issuers or Root Certificate Issuers
+   must supply the following additional information:
+
+   * URL of the current qualifying audit report.
+   * Links or references to issued end-entity certificates that demonstrate
+     them being treated as valid by a Certificate Consumer Member.
+
+   Such Applicant SHALL become a Member once the SMCWG has determined by
+   consensus among the Members during a SMCWG Meeting or Teleconference that
+   the Applicant meets all of the requirements above or, upon the request of
+   any Member of the SMCWG, by a Ballot among Members of the SMCWG. Acceptance
+   by consensus shall be determined or a Ballot of the Members shall be held
+   as soon as the Applicant indicates that it has presented all information
+   required above and has responded to all follow-up questions from the SMCWG
+   and the Member has complied with the requirements of Bylaw 5.5.
+
+   Certificate Issuer applicants that are not actively issuing S/MIME
+   certificates but otherwise meet these membership criteria MAY request to the
+   SMCWG that they be granted an invitation for Associate Member status in
+   accordance with Bylaw 3.1, subject to conditions designated by the SMCWG.
+
+2. Existing CAB Forum Members seeking to participate in the SMCWG, in accordance
+   to Bylaw 5.3.1(c), MUST formally declare their intent to participate in
+   writing and provide the SMCWG Chair with this declaration and evidence that
+   they meet the criteria set forth above. Such Applicants SHALL become Members
+   of the SMCWG as determined by consensus during a SMCWG Meeting or
+   Teleconference, or upon the request of any Member of the SMCWG, by a Ballot
+   among Members of the SMCWG.
+
+   In order to determine the composition of the initial set of SMCWG Members,
+   at least twenty-four (24) hours prior to the initial meeting of the SMCWG,
+   the SMCWG Chair SHALL publish a list of Members seeking to participate who
+   they determine meet the criteria set forth above. As the first order of
+   business at the first meeting of the SMCWG, those organizations on the
+   Chair’s list of proposed, qualifying Members SHALL vote to determine the
+   initial set of SMCWG Members.
```

Comment:
If the voting changes (discussed below) are adopted, this can be updated to say:

---

To confirm the list of initial Members, at least two-thirds of the organizations on the Chair's list of
proposed, qualifying Members SHALL publicly indicate vote in favor of adopting the list of Members. If the initial list fails to be adopted, the Chair SHALL consider the feedback and MAY update the initial list of proposed, qualifying Members as a result, and a second vote using these same rules shall be performed. If an initial list of Members cannot be agreed upon, the SMCWG shall be dissolved.

---

Basically, there's nothing in here that describes what happen if members don't vote in favor, and it suggests that a quorum for votes is only three votes. This attempts to make it so that 2/3 of those who express interest actually have to show up and vote. Similarly, by both public vote and through public Chair modification of the list, any shenanigans (e.g. such as removing all those with dissenting votes) are at least public.